### DISCUSSION TITLE
Can I have different things under the same unique identifier when style training?

### DISCUSSION THREAD

**Discussion by gilroff** [TheLastBen/fast-stable-diffusion#1320] [Q&A]
im training a culture style. so, can i have 4 instance images of ornaments, 4 of buildings from that culture, 4 images of landscape under the same unique identifier, if i turn on the style training option? I will also add sufficient concept images. also, any tips for instance images that look different?

**Comment by TheLastBen** [comment]
You can use the same identifier but the instance images must have something in common (filter, colors ...etc) to allow the model to understand the pattern.
The style_training option must be used only after initially training the model normally, it helps enhancing the existing trained subject without too much overfitting.

**Reply by gilroff** [reply]
But I should turn on the style training if Im training a style even for the first time, right? can you explain a bit more?

**Reply by TheLastBen** [reply]
if you tick the box style training, it won't pass the identifier to the unet, so you will have a hard time trying to summon the style if the model doesn't know its identifier, that's why you should only tick that box after the model has associated the instance images to the identifier.
